scripts/ocp_cleanup.py

```diff
@@ -0,0 +1,95 @@
+import argparse
+import sys
+
+from cfme.containers.provider.openshift import OpenshiftProvider
+from cfme.utils.providers import list_providers, ProviderFilter
+from cfme.utils.log import create_sublogger, add_stdout_handler
+
+logger = create_sublogger('ocp_cleanup')
+add_stdout_handler(logger)
+
+global_scc_names = ['anyuid', 'privileged']
+
+default_namespace_regex = ['^*?-project-',
+                           '^jenkins-*',
+                           '^*?s-appl-',
+                           '^test',
+                           '^long-test',
+                           '^external-test']
+
+
+def parse_cmd_line():
+    parser = argparse.ArgumentParser(argument_default=None)
+    parser.add_argument('-s', '--cleanup-scc', action='store_true',
```

Comment:
**QUESTION**
This argument is not used here, isn't it?

Reply:
good catch. there were more cleanup functions but I removed them along with args. 
I'll add this argument back to script